Flat/interface/PandaAnalyzer.h

```diff
@@ -60,7 +60,8 @@ public :
      kGenBosonPt =(1<<10),
      kVHBB       =(1<<11),
      kLepton     =(1<<12),
-     kLeptonFake =(1<<13)
+     kLeptonFake =(1<<13),
+     applyMCTriggers=(1<<14)
```

Comment:
This shouldn't be a preselection bit. It should be a member of the `Analysis` object. You want to record triggers in MC even if you're not preselecting on things.

Reply:
Let me change that. Then, it should have its own set function, right?

Reply:
No, the `Analysis` object has public members. They are set per-analysis in `Flat/python/analysis.py`

Reply:
Well, I have added it as a private member, I hope it's fine (see later)

Reply:
But then how do you plan to configure it in the job? The `Analysis` class has no private members.

Reply:
I simple add:
    skimmer.SetApplyMCTriggers(True)
if wanted. I believe your suggestion is just
skimmer.applyMCTriggers=True which is not very different